contracts/CO30_Matias_AnOldFriendsIdea.Altis/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,3 @@
+There is an ongoing civil war in Altis and during a recent raid on an AAF the FIA managed to secure two server units that contain sensitive information.
+
+For the time being they have been unable to crack the encryption, but this will change sooner or later. Therefore Synixe have been hired to move in on the FIA controlled city of Pyrgos and secure both of the server units.
```

Comment:
```suggestion
There is an ongoing civil war in Altis and during a recent raid on an AAF position, the FIA managed to secure two server units that contain sensitive information.

For the time being they have been unable to crack the encryption, but this will change sooner or later. Therefore Synixe have been hired to move in on the FIA controlled city of Pyrgos and secure both of the server units.
```